spec/board_spec.rb

```diff
@@ -36,4 +36,21 @@
     end
   end
 
+  describe '#valid_placement?' do
+    
+    it 'checks if the placement is equal to length of ship' do
+      board = Board.new
+      cruiser = Ship.new("Cruiser", 3)
+
+      expect(board.valid_placement?(cruiser, ["A1", "A2"])).to be false
+    end
+
+    it 'checks if coordinates are consecutive' do
```

Comment:
Maybe we should rename this to make sure it's clear this is the case where the ship is horizontal but the coords are not consecutive?

Reply:
Will complete this in the next round of refactoring. It's on the list.